### PR TITLE
feat(v2): bootstrap theme blog post page

### PR DESCRIPTION
**Pull request by fanny**
## Motivation

Add a component to render the blog post page

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

👍 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

FA5DE219-3EF2-4E0E-AB5B-24F640E6B3B7

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: packages/docusaurus-init/templates/bootstrap/blog/2018-09-11-Towards-Docusaurus-2.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
```
Comment (yangshun) [inline packages/docusaurus-init/templates/bootstrap/blog/2018-09-11-Towards-Docusaurus-2.md:1]: We should keep templates light and easy to tell it's a template. Come up with something else?

File: packages/docusaurus-init/templates/bootstrap/blog/2020-01-07-tribute-to-endi.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (yangshun) [inline packages/docusaurus-init/templates/bootstrap/blog/2020-01-07-tribute-to-endi.md:1]: Definitely should not include this in the template.

File: packages/docusaurus-init/templates/bootstrap/docusaurus.config.js [hunk 3]
```diff
@@ -54,5 +54,14 @@ module.exports = {
       copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
     },
   },
-  presets: [['@docusaurus/preset-bootstrap']],
+  presets: [
+    [
+      '@docusaurus/preset-bootstrap',
+      {
+        blog: {
+          editUrl: 'https://github.com/facebook/docusaurus/edit/master',
```
Comment (yangshun) [inline packages/docusaurus-init/templates/bootstrap/docusaurus.config.js:62]: Follow the one in `templates/classic/docusaurus.config.js`.

**Comment by fanny** [comment]
I didn't implement pagination, but i'll, on the next pr.

**Pull request merged**